src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj

```diff
@@ -411,6 +411,28 @@
       </VSIXSourceItem>
     </ItemGroup>
   </Target>
+  <!-- This is a copy of the Microsoft.VisualStudio.SDK.EmbedInteropTypes NuGet package, but only the list of
+       assemblies that we need. The package includes things like EnvDTE which are reasonable for consumers, but
```

Comment:
The 2nd sentence ("The package includes things like EnvDTE...") doesn't seem appropriate to our project.